### DISCUSSION TITLE
Allow organization admins to create deployments from the Vercel dashboard

### DISCUSSION THREAD

**Discussion by brianlovin** [vercel/community#7398] [Feature Question] [answered]
### Question

We've had a pretty frustrating experience using Vercel over the past few weeks, and I was told that opening this discussion would elevate this feature request for the Vercel team. Here's the gist:

1. Vercel doesn't allow new deployments from the Vercel dashboard if the last commit to main is from someone whose GitHub account is not also connected to the Vercel organization. See this error:

![Screen Shot 2024-07-03 at 21 32 29@2x](https://github.com/vercel/community/assets/1923260/d29c9b14-f649-4d62-bdd3-17c60fd9678b)

2. The last commit was created from a member of my Vercel organization, and a contributor to the GitHub repository. However, they also have a personal Vercel account for side projects which is linked to their GitHub account. **Vercel does not allow a GitHub account to be connected to two Vercel accounts**. 
3. As a result of this, any time this person on my team reconnects their GitHub account to their personal Vercel account, it becomes disconnected from their Work Vercel account. If their commit happens to be the last one in the repository, I'm unable to create a deployment from the Vercel dashboard.

This has become increasingly painful over the last few weeks where on multiple occasions I have been unable to create deployments. This might happen on my phone, away from my computer, which requires alerting my team to either force a new commit or have my team member scramble to reconnect their GitHub account to their Work Vercel account.

Please allow some combination of the following:

1. If I am the owner of the Vercel organization and the Owner of the GitHub organization, I should be able to create deployments from the Vercel dashboard without any restrictions.
2. A user should be able to associate a GitHub account with multiple Vercel accounts in order to support a personal side project + day job workflow.
3. Vercel organization admins should be able to override the error message/restriction above to force a new deployment off `main`

Thank you for reading!

**Comment by leerob** [comment, marked as answer]
Hey Brian! Question on this:

> However, they also have a personal Vercel account for side projects which is linked to their GitHub account. Vercel does not allow a GitHub account to be connected to two Vercel accounts.

Is there any reason why they don't have a single Vercel account? Previously, Vercel accounts didn't have the ability to have multiple email address. However, [that is now possible](https://vercel.com/changelog/accounts-can-now-have-multiple-email-addresses). You can think about your Vercel account like GitHub – you have one account and can be part of multiple organizations. But you can also have personal repos.

This means you can have personal side projects + your Vercel Campsite team, all in one Vercel account. Does that work here?

**Reply by brianlovin** [reply]
Thank you very much for replying so quickly! I'm going to see if we can consolidate Vercel accounts, I'm not sure we knew this was possible!

[a comment by an automated account is not shown]
